agent/api/eni/eni_test.go

```diff
@@ -27,6 +27,17 @@ const (
 	defaultDNS         = "169.254.169.253"
 	customDNS          = "10.0.0.2"
 	customSearchDomain = "us-west-2.compute.internal"
+
+	ipv4            = "1.2.3.4"
+	ipv4Gw          = "1.2.3.1"
+	ipv4Block       = "/20"
+	ipv4BlockAddr   = "1.2.0.0"
+	ipv4WithBlock   = ipv4 + ipv4Block
+	ipv4GwWithBlock = ipv4Gw + ipv4Block
+	ipv6            = "abcd:dcba:1234:4321::"
+	ipv6Block       = "/64"
+	ipv6BlockAddr   = "abcd:dcba:1234:4321::"
+	ipv6WithBlock   = ipv6 + ipv6Block
```

Comment:
These variable names could be better. ipv4Addr, ipv4Gw, ipv4SubnetPrefixLength, ipv4Subnet, ipv4AddrWithPrefixLength, ipv4GwWithPrefixLength, etc. Basically be consistent with names of the functions you are testing these variables against starting at line 95.

Not super important, but attention to detail is appreciated.

Reply:
that makes sense. updated these names